cluster/test-deploy/gcp-crio/vars.yaml

```diff
@@ -1,4 +1,36 @@
 ---
+provision_custom_repositories: [
```

Comment:
@smarterclayton for some reason, when we provision a new cluster (`make WHAT ....`), these repositories here are entirely skipped as if the `vars.yaml` file isn't taken into account _at all_. Cluster setup fails then as no dockertested/criotested repo will be on the system.